ext/standard/net.c

```diff
@@ -0,0 +1,293 @@
+/*
+   +----------------------------------------------------------------------+
+   | PHP Version 7                                                        |
+   +----------------------------------------------------------------------+
+   | Copyright (c) 1997-2017 The PHP Group                                |
+   +----------------------------------------------------------------------+
+   | This source file is subject to version 3.01 of the PHP license,      |
+   | that is bundled with this package in the file LICENSE, and is        |
+   | available through the world-wide-web at the following url:           |
+   | http://www.php.net/license/3_01.txt                                  |
+   | If you did not receive a copy of the PHP license and are unable to   |
+   | obtain it through the world-wide-web, please send a note to          |
+   | license@php.net so we can mail you a copy immediately.               |
+   +----------------------------------------------------------------------+
+   | Authors: Sara Golemon <pollita@php.net>                              |
+   +----------------------------------------------------------------------+
+ */
+
+#include "php.h"
+#include "php_network.h"
+
+#if HAVE_ARPA_INET_H
+# include <arpa/inet.h>
+#endif
+
+#if HAVE_NET_IF_H
+# include <net/if.h>
+#endif
+
+#if HAVE_GETIFADDRS
+# include <ifaddrs.h>
+# include <netdb.h>
+#endif
+
+#ifdef PHP_WIN32
+# include <intrin.h>
+# include <winsock2.h>
+# include <ws2ipdef.h>
+# include <Ws2tcpip.h>
+# include <iphlpapi.h>
+#endif
+
+zend_string* php_inet_ntop(const struct sockaddr *addr) {
```

Comment:
This is missing PHPAPI as you have declared in the header

Reply:
Normally the first declaration is decisive, but would be cleaner to repeat same, yep.

Thanks.

Reply:
As @weltling stated, not needed at point of definition when the included declaration has it, but sure, hurts nothing to add it.